src/OpenFeatureSDK/Model/StructureBuilder.cs

```diff
@@ -0,0 +1,156 @@
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+        /// <summary>
+        /// Set the key to the given double.
+        /// </summary>
+        /// <param name="key">The key for the value</param>
+        /// <param name="value">The value to set</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Set(string key, double value)
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+        /// <summary>
+        /// Set the key to the given long.
+        /// </summary>
+        /// <param name="key">The key for the value</param>
+        /// <param name="value">The value to set</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Set(string key, long value)
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+        /// <summary>
+        /// Set the key to the given bool.
+        /// </summary>
+        /// <param name="key">The key for the value</param>
+        /// <param name="value">The value to set</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Set(string key, bool value)
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+        /// <summary>
+        /// Set the key to the given <see cref="Structure"/>.
+        /// </summary>
+        /// <param name="key">The key for the value</param>
+        /// <param name="value">The value to set</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Set(string key, Structure value)
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+        /// <summary>
+        /// Set the key to the given DateTime.
+        /// </summary>
+        /// <param name="key">The key for the value</param>
+        /// <param name="value">The value to set</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Set(string key, DateTime value)
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+        /// <summary>
+        /// Set the key to the given list.
+        /// </summary>
+        /// <param name="key">The key for the value</param>
+        /// <param name="value">The value to set</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Set(string key, IList<Value> value)
+        {
+            this.Set(key, new Value(value));
+            return this;
+        }
+
+
+        /// <summary>
+        /// Remove the specified key.
+        /// </summary>
+        /// <param name="key">The key to remove</param>
+        /// <returns>This builder</returns>
+        public StructureBuilder Remove(string key)
+        {
+            this._attributes.Remove(key);
+            return this;
+        }
```

Comment:
Is this method needed now? Is is realistic that an builder would add and then remove something before having `build()` called on it?

Reply:
Removed the Removes.